src/lib/components/layout/FormLayout/README.mdx

```diff
@@ -287,91 +289,112 @@ This is a demo of all components supported by FormLayout.
       },
     ];
     return (
-      <FormLayout fieldLayout="horizontal" labelWidth="auto">
-        <>
+      <>
+        <div className="mb-6">
+          <ButtonGroup>
+            <Button
+              clickHandler={() => setFieldLayout('vertical')}
+              label="Vertical layout"
+              variant={fieldLayout === 'vertical' ? 'dark' : 'primary'}
+            />
+            <Button
+              clickHandler={() => setFieldLayout('horizontal')}
+              label="Horizontal layout"
+              variant={fieldLayout === 'horizontal' ? 'dark' : 'primary'}
+            />
+          </ButtonGroup>
+        </div>
+        <FormLayout fieldLayout={fieldLayout} labelWidth="auto">
+          <>
+            <TextField
+              changeHandler={() => {}}
+              id="form-layout-horizontal-first-name"
+              label="First Name"
+            />
+            <TextField
+              changeHandler={() => {}}
+              id="form-layout-horizontal-last-name"
```

Comment:
The attribute `id` value should not contain the word `horizontal` now that the `FormLayout.fieldLayout` is configurable.